packages/query-engine/source/controllers/query-engine-controller.ts

```diff
@@ -1,47 +1,86 @@
 import type { MySQLConfiguration, Resource, Query } from "@hypertool/common";
 
-import mysql from "mysql2/promise";
 import { QueryTemplateModel, ResourceModel } from "@hypertool/common";
-import { NextFunction } from "express";
+import { Knex, knex } from "knex";
 
 import type { ExecuteParameters } from "../types";
 
-const execute = async (
+const executeMySQL = async (
```

Comment:
Since we are using Knex, why do you need a specific function for MySQL? Is there a way to make this function generic?